charts/camunda-platform-alpha/templates/identity/configmap.yaml

```diff
@@ -213,6 +223,43 @@ data:
     {{- if .Values.global.identity.auth.enabled }}
     {{- if eq .Values.global.identity.auth.type "KEYCLOAK"}}
     keycloak:
+      url: {{ include "identity.keycloak.url" . | quote }}
+      setup:
+        user: {{ include "identity.keycloak.authAdminUser" . | quote }}
+        password: ${VALUES_KEYCLOAK_SETUP_PASSWORD:}
+      init:
+        console:
+          secret: ${VALUES_KEYCLOAK_INIT_CONSOLE_SECRET:}
+        operate:
+          secret: ${VALUES_KEYCLOAK_INIT_OPERATE_SECRET:}
+        tasklist:
+          secret: ${VALUES_KEYCLOAK_INIT_TASKLIST_SECRET:}
+        optimize:
+          secret: ${VALUES_KEYCLOAK_INIT_OPTIMIZE_SECRET:}
+        webmodeler:
+          root-url: {{ tpl .Values.global.identity.auth.webModeler.redirectUrl $ | quote }}
+        zeebe:
+          name: "Zeebe"
+      # Clients are passed as environment variables.
+      clients:
+      users:
```

Comment:
I don't think these firstUser options actually exist.  I can test this in an installation once I'm done understanding the diff.

Reply:
After installing the helm chart myself on this branch, I see that the firstUser settings still appear to work and that I can login with jesse/jesse instead of demo/demo. so I will resolve this thread.